src/components/Dropdown/Dropdown.tsx

```diff
@@ -252,6 +252,7 @@ export const Dropdown: React.FC<IDropdownProps> = (
         <div
             className={classnames(className)}
             style={{maxWidth}}
+            data-testid="moonstone-dropdown"
```

Comment:
As the purpose of this attribute is to add a selector for test, I propose adding it only on the test.

Reply:
Good point! It's not necessary for it to be in the component itself. I'll move it to the test.